ui/page/governance/splash_screen.go

```diff
@@ -43,15 +43,7 @@ func (pg *Page) splashScreenLayout(gtx layout.Context) layout.Dimensions {
 							}.Layout(gtx, txt.Layout)
 						}),
 						layout.Rigid(func(gtx C) D {
-							text := `<span style="text-color: gray">
-							The Decred community can participate in proposal discussions 
-							for new initiatives and request funding for these initiatives. 
-							Decred stakeholders can vote if these proposals should be approved 
-							and paid for by the Decred Treasury.
-
-							Would you like to fetch and view the proposals?
-						</span>`
-
+							text := values.StringF(values.StrGovernanceInfo, `<span style="text-color: gray">`, `<br>`, `</span>`)
```

Comment:
this seems like it would produce an error on the UI

Reply:
![Screenshot from 2022-05-26 17-11-42](https://user-images.githubusercontent.com/25265396/170548359-5a484c4f-fa2d-415b-89e1-972a36304933.png)